python/ray/train/callbacks/results_prepocessors/keys.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from typing import List, Dict, Optional, Iterable
+
+from ray.train.callbacks.results_prepocessors.preprocessor import \
+    ResultsPreprocessor
+from ray.train.constants import PROFILER_KEY
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_EXCLUDED_KEYS = [PROFILER_KEY]
+
+
+class KeysResultsPreprocessor(ResultsPreprocessor):
+    """Preprocesses results by key."""
```

Comment:
Can you add more description here about the arguments and a docstring for the methods?